### PR TITLE
Update de_DE (kinda big) and fix a single char in en_US

### PR DESCRIPTION
**Pull request by Vrontis**
Well... Huge numbers with different dividers, I guess?

### CODE REVIEW

File: src/main/resources/assets/languages/de_DE.json [hunk 1]
```diff
@@ -146,7 +149,7 @@
       "bait": "Der Fisch hat deinen Köder geschluckt!",
       "missing_equipped": "%1$sDu hattest eine %2$s Angelrute ausgerüstet, aber keine Angelrute der Art im Inventar! Die ausgerüstete Angelrute wurde zurückgesetzt.",
       "broken_drop": "%1$sDeine Angel ist kaputt gegangen, aber du kannst sie reparieren. Kaputten Gegenstand erhalten: %2$s *%3$s*",
-      "no_rod_equipped": "%1$sDu hast keine Angelrute ausgerüstet. Um eine auszurüsten, nutze `~>equip <Angelname>`, um dein Abenteuer zu starten!`"
+      "no_rod_equipped": "%1$sDu hast keine Angelrute ausgerüstet. Rüste eine mit ~>equip <Angelname>` aus, um dein Abenteuer zu beginnen!"
```
Comment (Xernium) [inline src/main/resources/assets/languages/de_DE.json]: Angelname vs Typ? Otherwise 👍

File: src/main/resources/assets/languages/de_DE.json [hunk 2]
```diff
@@ -593,7 +596,7 @@
         "key_locked": "%1$sDer Nutzer, den du versucht hast zu beanspruchen, ist verschlossen und kann nicht beansprucht werden.",
         "not_enough_money": "%1$sDu hast nicht genug Geld, um diese Waifu zu beanspruchen **(Kosten: %2$s Credits)**",
         "not_enough_slots": "%1$sDu hast nicht genug Plätze verfügbar, um diese Waifu zu beanspruchen. (Plätze: %2$s, Benutzt: %3$s)",
-        "success": "%1$s`%2$s` erfolgreich für %3$,d Credits beansprucht. Plätze benutzt: %4$s",
+        "success": "%1$s`%2$s` erfolgreich für %3$s Credits beansprucht. Plätze benutzt: %4$s",
```
Comment (Xernium) [inline src/main/resources/assets/languages/de_DE.json]: Benutzte Plätze sounds a bit more casual maybe? No idea

File: src/main/resources/assets/languages/de_DE.json (hunk 1 above)
Comment (Vrontis) [reply src/main/resources/assets/languages/de_DE.json]: Angelart maybe?

**Comment by Kodehawa** [comment]
ís this done

**Comment by Vrontis** [comment]
As long as I didn't miss anything, it should be done for now.
I tried to keep it updated, while you were still making fixes.

**Pull request merged**